README.md

```diff
@@ -21,21 +21,25 @@ docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
 
 ### Non-docker
 
-```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
-```
+Download binary from [releases](releases) for you platform.
```

Comment:
This link works? Or just links to a folder called `releases`

Reply:
will fix